pkgs/development/python-modules/catppuccin/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "catppuccin";
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-mHNuV3yIuFL2cixDOr+//+/b9iD2fN82cfLzZkegxKc=";
+  };
+
+  propagatedBuildInputs = [ pygments ];
+
+  doCheck = true;
+  pythonImportsCheck = [ "catppuccin "];
```

```suggestion
  pythonImportsCheck = [ "catppuccin"];
```

I assume this was a typo too

---

Indeed. Thanks.